packages/medusa/src/loaders/module-definitions/index.ts

```diff
@@ -10,15 +10,36 @@ export default ({ modules }: ConfigModule) => {
   for (const definition of MODULE_DEFINITIONS) {
     let resolutionPath = definition.defaultPackage
 
+    const mod = projectModules[definition.key]
```

Comment:
todo: Change `mod` to `module`

Reply:
think module is a reserved keyword

Reply:
Ah yeah, might be. Would do `projectModule` then :)

Reply:
not reserved, but I have changed it to `moduleConfiguration` anyways to avoid confusion (even vscode changed the color of the variable-name thinking it was a keyword).